### PR TITLE
modify result page

### PR DESCRIPTION
**Pull request by fadeomar**
**there is some misconceptions in the result page :** 
  - problem : What if there are two types with the same rating
     - Each type is handled individually so that each type is called and then displayed to the user
  - problem : What if the requested type has no database results
    - This case is treated as the filter case, the first five are called in the data base
  - usage : Use FUNCTION tofind for the highest evaluation of the type of therapist according to the data coming from the questionnaire page.
  - usage : Use Function to bring the data according to the every therapy type 
  - fix : propsType problems 
  - fix : eslint problems 
  - fix : arrange the code according to the conventional method

### CODE REVIEW

File: client/src/components/App/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
-import { Helmet } from 'react-helmet';
+import Helmet from 'react-helmet';
```
Comment (AbdallahAmmar96) [inline client/src/components/App/index.js]: check if it's still working

File: client/src/components/pages/Results/index.js [hunk 2]
```diff
@@ -37,33 +73,47 @@ export default class ResultsPage extends React.Component {
   };
 
   render() {
-    // eslint-disable-next-line react/destructuring-assignment
     const { therapist, type } = this.state;
     return (
-      <>
+      <Fragment>
```
Comment (AbdallahAmmar96) [inline client/src/components/pages/Results/index.js]: you don't need a fragment here see the documentation

File: client/src/components/pages/Results/index.js [hunk 3]
```diff
@@ -1,31 +1,67 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```
Comment (AbdallahAmmar96) [inline client/src/components/pages/Results/index.js]:
```suggestion
import React from 'react';
```

File: client/src/components/App/index.js (hunk 1 above)
Comment (fadeomar) [reply client/src/components/App/index.js]: it's working without Carly brackets

File: client/src/components/pages/Results/index.js (hunk 2 above)
Comment (fadeomar) [reply client/src/components/pages/Results/index.js]: yeah I remove it

File: client/src/components/pages/Results/index.js (hunk 3 above)
Comment (fadeomar) [reply client/src/components/pages/Results/index.js]: delete Fragment form importing

**Pull request closed without merging**